### DISCUSSION TITLE
SetUp notification to a bucket for  minio in k8s pod

### DISCUSSION THREAD

**Discussion by adeesh-vunet** [minio/minio#17295] [Q&A] [answered]
I have setup minio in a k8s pod and did port forwarding to access it from outside. 
I have used golang to interact with minio. 
I'm able to do operation like : create bucket, put object, etc.  But I'm not able to add notification using [SetBucketNotification](https://min.io/docs/minio/linux/developers/go/API.html#setbucketnotification-ctx-context-context-bucketname-string-config-notification-configuration-error) API, because it is excepting a valid ARN, I'm not sure what would be the ARN in my case here or does **SetBucketNotification** support minio set up like mine. 

Can some one provide some ideas on how to use **SetBucketNotification**, API in my case ?

**Comment by harshavardhana** [comment, marked as answer]
> I'm able to do operation like : create bucket, put object, etc. But I'm not able to add notification using [SetBucketNotification](https://min.io/docs/minio/linux/developers/go/API.html#setbucketnotification-ctx-context-context-bucketname-string-config-notification-configuration-error) API, because it is excepting a valid ARN, I'm not sure what would be the ARN in my case here or does **SetBucketNotification** support minio set up like mine.

Looks like you have not followed our docs https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html
